resources/Dockerfile

```diff
@@ -1,7 +1,16 @@
-FROM golang:1.17-alpine as builder
+FROM golang:1.18-alpine as builder
```

Comment:
I remember I updated this yesterday in #18, anyway, not sure why have this diff..

Reply:
sorry, I think cuz I forgot to trace the upstream.